transport/tchannel/config.go

```diff
@@ -35,25 +35,27 @@ import (
 // TransportConfig configures a shared TChannel transport. This is shared
 // between all TChannel outbounds and inbounds of a Dispatcher.
 //
-//  transports:
-//    tchannel:
-//      connTimeout: 500ms
-//      connBackoff:
-//        exponential:
-//          first: 10ms
-//          max: 30s
+//	transports:
+//	  tchannel:
+//	    connTimeout: 500ms
+//	    connBackoff:
+//	      exponential:
+//	        first: 10ms
+//	        max: 30s
+//	  enableMPTCP: true
```

Comment:
indentation off: enableMPTCP should be on the same level as `connTimeout` and `connBackoff`